pages/components/Navbar/Layout.tsx

```diff
@@ -0,0 +1,14 @@
+import Navbar from './Navbar'

```

Comment:
can you explain what is this `Layout` component for ?

Reply:
Higher order component

Reply:
what is it for ?

Reply:
High order component of navbar

Reply:
should i change the name?